gemrb/core/Scriptable/Scriptable.cpp

```diff
@@ -2038,6 +2044,7 @@ int Movable::GetPathLength()
 PathNode *Movable::GetNextStep(int x)
 {
 	if (!step) {
+		error("GetNextStepWIP", "GetNextStep hit with step = null");
 		DoStep((unsigned int) ~0);
```

Comment:
analyzers will probably complain about unreachable code
```suggestion

```

Reply:
This might be changed to, like, not die when hit with step == NULL, and just return NULL?

Reply:
the point was that this isn't supposed to be hit and we didn't know why it was needed before. error() is our release-build friendly assert. I think we can just  replace the if-block with a normal assert once some more gameplay testing is done. Or to be on the same side, just return null, like you say, since we're right before a big release.

Reply:
Okay very well, removing the unreachable code and leaving the error, maybe I can do some post-merge play-testing myself too.